go.mod

```
@@ -18,7 +18,7 @@ require (
 	github.com/prometheus/client_golang v1.2.1 // indirect
 	github.com/spf13/cobra v0.0.7
 	github.com/spf13/pflag v1.0.5
-	github.com/submariner-io/shipyard v0.0.0-20200416084120-97f62b652ea2
+	github.com/submariner-io/shipyard v0.0.0-20200415131458-43e0c8dc8ea3
```

This needs to be recent enough to have the e2e framework fixes:

```suggestion
	github.com/submariner-io/shipyard v0.0.0-20200423151725-235cee4d4bda
```